entities/author.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'ffaker'
+require './tasks/validate'
+
+# authors of books
+class Author
+  include Validate
+  attr_reader :name, :biography
+
+  def initialize(name, biography)
```

Comment:
biography is optional
https://docs.google.com/document/d/1z5-iLgDTvxLW3uZmTylxVg4ASClxQn90MjZx_Y8-LaQ/edit

Reply:
done